ts/sagas/startup/loadServiceDetailRequestHandler.ts

```diff
@@ -100,3 +106,109 @@ export function* watchServicesDetailLoadSaga(
     );
   }
 }
+
+/**
+ * listen for loading services details events to extract some track information
+ * like amount of details to load and how much time they take
+ */
+function* watchLoadServicesDetailToTrack() {
+  yield takeLatest(
+    [loadServicesDetail, loadServiceDetail.success, applicationChangeState],
+    action => {
+      switch (action.type) {
+        // request to load a set of services detail
+        // copying object is needed to avoid "immutable" error on frozen objects
+        case getType(loadServicesDetail):
+          const stats: ServicesDetailLoadTrack = {
+            ...servicesDetailLoadTrack,
+            kind: undefined,
+            startTime: new Date().getTime() as Millisecond,
+            servicesId: new Set([...action.payload]),
+            loaded: 0,
+            toLoad: servicesDetailLoadTrack.servicesId.size
```

Comment:
```suggestion
            toLoad: action.payload.length
```
With the current implementation `toLoad` is always `0`.

Reply:
that's a bug, due to refactoring!
Good catch!
fixed 
https://github.com/pagopa/io-app/pull/2785/commits/29baf38140075c7d4de02712159045a00e7861d6